Dockerfile

```
@@ -1,4 +1,4 @@
-FROM rust:1.67.0-alpine3.17 as builder
+FROM rust:1.72.0-alpine3.17 as builder
```

I think we can also bump the Alpine version.

```suggestion
FROM rust:1.72.0-alpine3.18 as builder
```